upload.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+USER=$1
+CLUSTER=$2
+DIRNAME=$3
+
+# any optional arguments you wish to pass directly to rsync go in position 4
+
+# exclude the hidden files, target, lib, project, src... except the assembled jar
+rsync $4 -v -a --include='target' --include='target/scala-2.10' --include='target/scala-2.10/*.jar' \
+   --include='spot-lda-c' --include='spot-lda-c/*'  --include='*.py'  --include='*.sh' --include 'top-1m.csv' \
+  --exclude='*' .  $USER@$CLUSTER:~/$DIRNAME
```

Comment:
on one hand:

people would almost certainly use it and appreciate it

on the other:

there is a process for adding stuff to the product and we didn't follow it
there's no documentation for it